lib/routes/display_name/post.js

```diff
@@ -30,7 +37,10 @@ module.exports = {
   },
   validate: {
     payload: {
-      displayName: Joi.string().required().allow('').regex(ALLOWED_DISPLAY_NAME_CHARS)
+      displayName: Joi.alternatives().try(
+        Joi.string().required().allow('').regex(ALLOWED_DISPLAY_NAME_CHARS),
+        Joi.string().required().allow('').regex(ALLOWED_EMOJI_CHARS)
+      )
```

Comment:
I don't understand the use of `try` here; IIUC this will either allow the same display-name chars as we did before, or it will allow display names composed *entirely* of emoij, but not something like `"embedded 👍 emoji"`.  Am I missing something?